packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -253,6 +256,7 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
         >
           <rect
             key={index}
+            id={getId('_MSBC_rect_')}
```

Comment:
You can call getId only once and append the key to it to make it unique.
Similar approach is followed in other charts